src/hexen/g_game.c

```diff
@@ -2101,6 +2109,11 @@ void G_DoPlayDemo(void)
     precache = true;
     usergame = false;
     demoplayback = true;
+
+    if (netgame == true)
```

Comment:
Please don't write `x == true`, it's the same as just `x`. Similarly `x == false` is just `!x`.

Reply:
I'm aware 😉 
I left it this way for consistency but I'm happy to change it.

Reply:
Updated